autoblocks/_impl/tracer.py

```diff
@@ -61,7 +50,8 @@ def __init__(
                 f"You must provide an ingestion_key or set the {AutoblocksEnvVar.INGESTION_KEY} environment variable."
             )
 
-        self._client = get_or_create_ingestion_client(ingestion_key)
+        self._client_headers = {"Authorization": f"Bearer {ingestion_key}"}
+        self._client = global_state.http_client()
```

Comment:
we could probably drop this and just call `global_state.http_client().post` directly in send_event? wdyt?

Reply:
I like that - no reason to obfuscate that its the global client